buildSrc/src/main/java/io/micronaut/guides/feature/MicronautRestAssured.java

```diff
@@ -0,0 +1,59 @@
+package io.micronaut.guides.feature;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.build.dependencies.MicronautDependencyUtils;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class MicronautRestAssured implements Feature {
```

Comment:
This file will be removed once 3.6.0 in guides. 

The build will fail because starter already will ship with a guide with a feature of the exact name.

https://github.com/micronaut-projects/micronaut-starter/pull/1288